src/apis/token_api.ts

```diff
@@ -1,18 +1,33 @@
 import * as Web3 from "web3";
-import { Web3Utils } from "../../utils/web3_utils";
-import { ContractsAPI } from "./";
+import * as singleLineString from "single-line-string";
 import { BigNumber } from "bignumber.js";
+
+import { Web3Utils } from "utils/web3_utils";
+
+import { ContractsAPI } from "./";
+import { Assertions } from "../invariants";
 import { TxData } from "../types";
 
 const TRANSFER_GAS_MAXIMUM = 70000;
 
+export const TokenAPIErrors = {
+    INSUFFICIENT_SENDER_BALANCE: () =>
```

Comment:
Let's include the sender's address in the message, i.e.:

`Sender with address ${address} does not ...`